tests/Unit/inc/Engine/Optimization/DeferJS/DeferJS/getExcluded.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare( strict_types=1 );
+
+namespace WP_Rocket\Tests\Unit\inc\Engine\Optimization\DeferJS\DeferJS;
+
+use Brain\Monkey\Functions;
+use Mockery;
+use WP_Rocket\Admin\Options_Data;
+use WP_Rocket\Engine\Optimization\DeferJS\DeferJS;
+use WP_Rocket\Tests\Unit\TestCase;
+
+/**
+ * @covers \WP_Rocket\Engine\Optimization\DeferJS\DeferJS::get_excluded
+ *
+ * @group  DeferJS
+ */
+class Test_GetExcluded extends TestCase {
+
+	private $options;
+	private $defer_js;
+
+	public function setUp(): void {
+		parent::setUp();
+
+		$this->options  = Mockery::mock( Options_Data::class );
+		$this->defer_js = new DeferJS( $this->options );
+	}
+
+	public function testShouldReturnEmptyWhenCannotDefer() {
+		$this->options->shouldReceive( 'get' )
+		              ->once()
+		              ->andReturn( false );
```

Comment:
I would add `->with( 'defer_all_js', 0 )` here too to validate the exact expectation for `get`, as it's a generic method